cumulus/polkadot-parachain/polkadot-parachain-lib/src/cli.rs

```diff
@@ -119,6 +119,12 @@ pub struct Cli<Config: CliConfig> {
 	#[command(flatten)]
 	pub run: cumulus_client_cli::RunCmd,
 
+	/// Use manual seal consensus.
+	///
+	/// This works only with dev chains (e.g. asset-hub-rococo-dev).
+	#[arg(long)]
+	pub manual_seal: bool,
```

Comment:
The raw manual seal also requires you to submit some kind of an RPC to author a block. What I have done in the past, I think this is a good improvement to it, is to run an RPC client internally that pings the manual seal server and produces blocks every x ms. 

This is then how I re-formulated this CLI argument as `--dev-block-time 100`, which produces a block every 100ms.